GeeksCoreLibrary/Modules/PostalServices/PostNL/Interfaces/IPostNLService.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Threading.Tasks;
+using GeeksCoreLibrary.Modules.PostalServices.PostNL.Models;
+using GeeksCoreLibrary.Modules.PostalServices.PostNL.Services;
+
+namespace GeeksCoreLibrary.Modules.PostalServices.PostNL.Interfaces
+{
+    public interface IPostNLService
+    {
+        /// <summary>
+        /// Creates a PostNL track and trace label using the specified order and PostNL API request
+        /// </summary>
+        /// <param name="orderId">The orderId of the order a label must be created for</param>
+        /// <param name="request">The API request response</param>
+        /// <returns>Object containing all the information about the order</returns>
+        public Task<ShipmentResponseModel> CreateTrackTraceLabel(string orderId, ShipmentRequestModel request);
```

Comment:
Async functies moeten altijd de suffix Async hebben, deze graag hernoemen naar `CreateTrackTraceLabelAsync`. In interfaces geen public/protected/private modifiers gebruiken, het woord `public` hier weghalen.